app/assets/stylesheets/common/base/edit-category.scss

```diff
@@ -0,0 +1,144 @@
+div.edit-category {
+  display: grid;
+  grid-template-columns: 1fr 5fr;
+  grid-template-rows: auto auto auto;
+  grid-row-gap: 1em;
+  grid-column-gap: 1.5em;
+  grid-template-areas: "sidebar header" "sidebar content" "sidebar footer";
+
+  .edit-category-title {
+    grid-area: header;
+    display: flex;
+    justify-content: space-between;
+    align-self: start;
+  }
+
+  .nav-stacked {
+    grid-area: sidebar;
+    grid-row: 1 / span 3;
+    background: transparent;
+  }
+
+  #list-area & h2 {
+    margin: 0;
+  }
+
+  section.field {
+    margin-bottom: 1em;
+  }
+
+  .edit-category-tab-general {
+    .category-chooser {
+      width: unquote("min(340px, 90%)");
+    }
+  }
+
+  .permission-selector {
+    margin-right: 0.25em;
+  }
+
+  .pending-permission-change-alert {
+    margin-left: auto;
+    max-width: 250px;
+    background: var(--primary-very-high);
+    color: var(--secondary);
+    margin-top: 10px;
+    padding: 5px 10px;
+    position: relative;
+    .arrow-div {
+      border: solid transparent;
+      content: " ";
+      position: absolute;
+      border-bottom-color: var(--primary-very-high);
+      border-width: 7px;
+      top: -13px;
+      left: 200px;
+    }
+  }
+
+  .add-permission {
+    position: relative;
+    top: 0.1em;
+  }
+
+  .edit-category-tab-settings {
+    $category-settings-width: unquote(
+      "min(500px, 90%)"
+    ); // Consistent width makes this all easier to read
+    > section {
+      margin-bottom: 1.5em;
+    }
+
+    input[type="number"] {
```

Comment:
if we can use this to drop any direct html element styling that would be cool :+1:

Reply:
Not sure I understand this either, would you prefer to target something other than `input[type]`?

Reply:
yes should style on class not html element

Reply:
Hmm, so, I started doing this, but then I stopped. It feels wrong to do it for just this one small part of the app. 

From a quick search, we use `input[type]` much more than we use classes to target inputs. If we device to go with classes instead, it should be a global refactor, that way it's consistent. 

(This did lead to some cleanup in the adjacent CSS though, that is nice.)